locales/en/app.ftl

```diff
@@ -3,11 +3,14 @@
 ## The following messages are brand and should be kept entirely in English
 ## unless otherwise indicated.
 
+-product-fx-monitor = Firefox Monitor
```

Comment:
We shouldn't add new terms if we already have them (this and Firefox).

If the concern is making sure they won't be removed when we clean up the file, then group them together with a clear warning.

Reply:
Thanks @flodolo – you're right on the concern.  Additionally the way the original variables were named is confusing/misleading (looks like you've also called this out RE "brand" below).

Question: I assume placement in the page does not matter?  Given your feedback, my plan is to move the new (duplicated) terms down into the "V2" section so that it's clear these are part of the new build.  Then, after cleanup/deletion of old terms, move them back to top-of-file with existing brand vars that remain.

Reply:
Yes, placement is not important, and it would be OK to move them around afterwards within the file.